app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class ConfirmationsController < ApplicationController
+  before_action :redirect_if_authenticated
+  def create
+    @user = User.find_by(email: params[:user][:email].downcase)
+
+    if @user.present? && @user.unconfirmed?
+      @user.send_confirmation_email!
+      redirect_to root_path, notice: 'Please check your email for confirmation instructions'
+    else
+      redirect_to new_confirmation_path, alert: 'We could not find a user with that email has already been confirmed'
```

Comment:
can we reconsider the alert text again?

Reply:
Bhaiya, do I change all of the alerts or just this controller?

Reply:
IMHO, Alert message should follow generic format, not that kind of specific. Thank you!